nekoyume/Assets/_Scripts/UI/Module/IAP/IAPShopProductCellView.cs

```diff
@@ -140,96 +151,86 @@ private void Refresh()
                     bottomLayout.spacing = 0;
                     break;
                 case "1x2":
-                    _rect.sizeDelta = new Vector2(_rect.sizeDelta.x, 467);// add spacing size
+                    _rect.sizeDelta = new Vector2(_rect.sizeDelta.x, 467); // add spacing size
                     bottomButtonLayoutElement.minHeight = 75;
                     bottomLayout.spacing = 3;
                     break;
-                default:
-                    break;
             }
 
+            var metadata = _purchasingData.metadata;
+            Debug.Log($"{metadata.localizedTitle} : {metadata.isoCurrencyCode} {metadata.localizedPriceString} {metadata.localizedPrice}");
+
             foreach (var item in price)
             {
-                item.text = MobileShop.GetPrice(_puchasingData.metadata.isoCurrencyCode, _puchasingData.metadata.localizedPrice);
+                item.text = MobileShop.GetPrice(metadata.isoCurrencyCode, metadata.localizedPrice);
             }
-            Debug.Log($"{_puchasingData.metadata.localizedTitle} : {_puchasingData.metadata.isoCurrencyCode} {_puchasingData.metadata.localizedPriceString} {_puchasingData.metadata.localizedPrice}");
 
+            var isDiscount = _data.Discount > 0;
             foreach (var item in discountObjs)
             {
                 item.SetActive(isDiscount);
             }
 
+            tagObj.SetActive(false);
+            discount.gameObject.SetActive(false);
+            timeLimitText.gameObject.SetActive(false);
             if (isDiscount)
             {
+                discount.text = $"{_data.Discount}%";
                 foreach (var item in preDiscountPrice)
                 {
-                    var originPrice = (_puchasingData.metadata.localizedPrice * ((decimal)100 / (decimal)(100 - _data.Discount)));
-                    var origin = MobileShop.GetPrice(_puchasingData.metadata.isoCurrencyCode, originPrice);
+                    var originPrice = metadata.localizedPrice * ((decimal)100 / (100 - _data.Discount));
```

Comment:
I think, `Discount` field may have set by '100'. then, It can throw DivideByZeroException.